dotcom-rendering/src/components/FrontSection.tsx

```diff
@@ -377,6 +408,15 @@ const secondaryLevelTopBorder = css`
 	}
 `;
 
+const carouselNavigationPlaceholder = css`
+	${until.leftCol} {
+		height: 44px;
```

Comment:
Is this the 'fixed minimum height placeholder' referred to in the PR description?